14_functions_lab/04.py

```diff
@@ -1,6 +1,14 @@
-"""
-Write a function that takes minlen and
-a list of words, and returns only the words
-longer than minlen
-"""
-
+_author_ = 'mariag'
+
+def longer_than(length ,*args):
+    if type(length) != int:
+            raise Exception ("longer_than expects first parameter integer")        
+    list_res = []
+    for word in args:
+        if type(word) != str:
+            raise Exception ("longer_than expects (int, [str])")
+        if len(word)>length:
+            list_res.append(word)
+    return list_res
+
+print longer_than(3,"she","loves","you","yeee","ye","yeee")
```

Comment:
👍 
Now try to do it with list comprehension (but without the error checking of strings)